### ISSUE TITLE
StatementOrder -  can't suppres issue

### ISSUE THREAD

**Issue by clement-brodu** [Riverside-Software/sonar-openedge#1120]
Hello,

Versions:  
- plugin 2.26.0
- openedge 12.8.2

The suppress preprocess does have effect for StatementOrder rules. 

I tried to write the suppress statement everywhere but it does not work:

![image](https://github.com/Riverside-Software/sonar-openedge/assets/37274615/bac173f1-0c85-4c6f-b213-dddf993912a8)

![image](https://github.com/Riverside-Software/sonar-openedge/assets/37274615/6da0a411-744d-40ff-afca-05a78c529b38)

```abl
{&_proparse_ prolint-nowarn(stmt_order)}
RUN foo.p.

{&_proparse_ prolint-nowarn(stmt_order)}
PROCEDURE proc1: {&_proparse_ prolint-nowarn(stmt_order)}
  // Content
END PROCEDURE.

{&_proparse_ prolint-nowarn(stmt_order)}
RUN bar.p. // Non-compliant 

PROCEDURE proc2:
  // Content
END PROCEDURE.
```

**Comment by gquerret** [comment]
Documentation is wrong, the correct directive is `{&amp;_proparse_ prolint-nowarn(statement_order)}`.
This code sample doesn't raise any issue:
```abl
RUN foo.p.

{&_proparse_ prolint-nowarn(statement_order)}
PROCEDURE proc1:
  // Content
END PROCEDURE.

RUN bar.p. // Non-compliant 

PROCEDURE proc2:
  // Content
END PROCEDURE.
```

Documentation has been updated.

**Comment by gquerret** [comment]
Had a look at the Git history of the documentation, that was never fixed 😲 I don't know what exactly happened...
Fix is now part of the develop branch, so it will be part of next release.
